packages/@lwc/synthetic-shadow/src/faux-shadow/slot.ts

```diff
@@ -105,11 +106,15 @@ defineProperties(HTMLSlotElement.prototype, {
         value(
             this: HTMLSlotElement,
             type: string,
-            listener: EventListener,
-            options?: boolean | AddEventListenerOptions
+            _listener: EventListener,
+            _options?: boolean | AddEventListenerOptions
         ) {
             // super.addEventListener - but that doesn't work with typescript
-            HTMLElement.prototype.addEventListener.call(this, type, listener, options);
+            HTMLElement.prototype.addEventListener.apply(this, ArraySlice.call(arguments) as [
```

Comment:
Why not addEventListener from `../env/element`?

Reply:
because that one is not patched, while `HTMLElement.prototype.addEventListener` is patched